appveyor.yml

```diff
@@ -17,8 +17,8 @@
 #   3. Override 'JENKINS_ARTIFACTORY_PASSWORD' with variable encryption.
 
 image:
-  - Ubuntu
   - Visual Studio 2017
+  - Ubuntu
```

Comment:
Is there a reason why the order was changed here? Otherwise I suggest to keep to original order.